static/templates/settings/organization-permissions-admin.handlebars

```diff
@@ -143,7 +143,7 @@
             </div>
 
             <div class="input-group disableable {{#unless realm_allow_message_editing}}control-label-disabled{{/unless}}">
-                <label for="realm_message_content_edit_limit_minutes"
+                <label for="id_realm_message_content_edit_limit_minutes"
```

Comment:
I think these ID fixes should be in a separate commit from the `aria-label` changes, since they seem to be addressing totally different issues.